plugins/woocommerce/src/Internal/DataStores/Orders/OrdersTableDataStore.php

```diff
@@ -1987,6 +1988,28 @@ public function delete_order_data_from_custom_order_tables( $order_id ) {
 		}
 	}
 
+	/**
+	 * Deletes shared order data (created with and without custom order tables in use).
+	 *
+	 * @param int $order_id The order ID.
+	 * @return void
+	 */
+	private function delete_order_data_from_shared_tables( $order_id ) {
```

Comment:
Could you use [this inherited method](https://github.com/woocommerce/woocommerce/blob/9be843422d6a10d699ca8aa59f5e79eadb99720f/plugins/woocommerce/includes/data-stores/abstract-wc-order-data-store-cpt.php#L422) instead of adding a new one?

Reply:
Good point, changed.